src-docs/src/views/comment/comment_example.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
```

Comment:
And then you just need to import `Link` from here:
```suggestion
import React, { Link } from 'react';
```